Classes/Utility/ClassCacheManager.php

```diff
@@ -27,11 +27,17 @@ class ClassCacheManager
      */
     protected $classCache;
 
+    /**
+     * @var array
+     */
+    protected $constructorParameters = [];
```

Comment:
@stigfaerch, what's that being used for? Dead code?

Reply:
Now removed